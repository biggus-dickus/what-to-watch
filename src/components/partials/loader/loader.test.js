import * as React from 'react';
import renderer from 'react-test-renderer';

import Loader from './loader';

const tree = renderer.create(<Loader show={true} />).toJSON();

it(`<Loader /> should render correctly`, () => expect(tree).toMatchSnapshot());
