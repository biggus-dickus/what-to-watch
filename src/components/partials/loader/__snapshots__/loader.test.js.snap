// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should render correctly 1`] = `
<div
  className="loader"
>
  <p
    className="visually-hidden"
  >
    Loadingâ€¦
  </p>
</div>
`;
